### PR TITLE
Handle merged SNP ids when resolving PAR regions

### PR DESCRIPTION
**Pull request by amoffet**
In my data set, I discovered that rs113378274  has been merged into rs72608386, which was not handled properly by snps.  These changes consider merged SNP ids.  
I have not been able to properly set up my dev environment yet - it's a managed laptop and the virus scanner is preventing me from running the tests collectively.  I'll get that fixed shortly.  Would somebody please run them for me? The test_snps.test_build_detected_PAR_snps test passes - which is the test I modified to check my changes.  Any feedback is welcome.

### CODE REVIEW

**Comment by amoffet** [comment]
I will fix these errors.

**Comment by apriha** [comment]
Thanks @amoffet for the PR and for handling this case! As you saw, Travis CI is running the tests.

File: CONTRIBUTORS.rst [hunk 1]
```diff
@@ -27,12 +27,14 @@ Listed in alphabetical order.
 ================ ===============
 Name             GitHub
 ================ ===============
+Alan Moffet `@amoffet`_
```
Comment (apriha) [inline CONTRIBUTORS.rst]: For the table to render, GitHub username needs to align with those below (i.e., add spaces before).

**Comment by amoffet** [comment]
I believe everything is good now.  I have a VM with a working environment so that I can continue to work on snps and lineage.  I'd like to begin work on implementing phasing like the kind in David Pike's utilities.

**Comment by castedo** [comment]
@amoffet Regarding David Pike's parent-child phasing ... if you want to also implement "grandparent phasing", checkout github.com/castedo/genomology. Some of that math and R code could maybe be ported into lineage.

File: tests/test_snps.py [hunk 2]
```diff
@@ -131,6 +131,8 @@ def test_build_detected_PAR_snps(self):
             snps = SNPs("tests/input/GRCh37_PAR.csv")
             assert snps.build == 37
             assert snps.build_detected
+            # rs113378274 was merged into rs72608386 and appears on the X chromosome
+            assert snps.snps.loc["rs113378274", "chrom"] == "X"
```
Comment (amoffet) [inline tests/test_snps.py]: This seems to be the wrong place for this.  Would you prefer a different test? or is the test below: test_remap_snps_37_to_38_with_PAR_SNP sufficient?

Comment (amoffet) [reply tests/test_snps.py]: I deleted this - as this test is responsible for testing the build detection.  If you would like another  test, let me know.

Comment (apriha) [reply tests/test_snps.py]: `test_remap_snps_37_to_38_with_PAR_SNP` is sufficient since it tests the end result.

**Comment by apriha** [comment]
Merging the PR. Thanks again @amoffet ! Any help you'd like to provide going forward would be awesome and greatly appreciated!

**Pull request merged**